src/platform/modulesmanager/tests/ModuleManagerTests/src/MpiTests.cpp

```diff
@@ -114,13 +114,11 @@ namespace Tests
     {
         std::string clientName = "MpiTests.MpiSetDesired";
         char payload[] = R""""(
-            [
-                {
-                    "TestComponent1": {
-                        "testParameter": "testValue"
-                    }
+            {
+                "TestComponent1": {
+                    "testParameter": "testValue"
                 }
```

Comment:
Can we have more complex test cases here? Multiple components and objects and also settings within objects. 

Maybe do something with one instance for every possible type? E.g. arrays of strings and integers for settings, single and array objects, etc.

Reply:
Agreed 👍🏻